lib/block/blockupload.js

```diff
@@ -84,7 +84,7 @@ async function* generateBlockUploadTransfer(options) {
         }
 
         const contentType = options.headers && options.headers['content-type'];
-        console.log(`Transfer asset to upload is of content type ${contentType}`);
+        console.log(`Transfer asset to upload is of content type ${contentType} and size ${uploadFile.fileSize} bytes`);
```

Comment:
what is there is no uploadFile.fileSize?

Reply:
It will be undefined/default, meaning underlying functions also used the default values (important information for my debugging)